src/controllers/main.ts

```diff
@@ -186,3 +193,66 @@ export const getHelpController = async (
     next(error);
   }
 };
+
+/**
+ * list the pages handled with the external CMS
+ *
+ * Record of <internalSlug, externalSlug>
+ */
+const EXISTING_EXTERNAL_PAGES: Record<string, string> = {
+  accessibilite: "moncomptepro-accessibilite",
+  confidentialite: "moncomptepro-politique-de-confidentialite",
+  "conditions-generales": "moncomptepro-conditions-generales-d-utilisation",
+};
+
+export const getLegalPageController = async (
```

Comment:
I'm sure I didn't handle errors that well. Waiting for your feedback about the whole feature before taking the time to polish it :)